web/templates/pages/members.html

```diff
@@ -20,13 +19,13 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
-					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
+					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="{{MEDIA_URL}}{{member.profile_picture.url}}" />{{member.fullname}}</a> (Mentor cat)<li>
```

Comment:
Django has a method on models with choices, which is:

`.get_FIELD_NAME_display`, so in your case you can use `member.get_activity_status_display` so you don't need `if` checkes. Also check `forloop.group` so you can group them by type.

In addition, it will be more readable if you use `{{ variable }}`, so with spaces inside.

It's also good practice to add `alt=` attribute to images. In your case `alt="{{ member.fullname }}"`